src/BubbleChart/BubbleSeries.tsx

```diff
@@ -78,17 +78,18 @@ export const BubbleSeries: FC<Partial<BubbleSeriesProps>> = ({
       >
         <CloneElement<BubbleProps>
           element={bubble}
-          id={`${id}-bubble`}
+          id={`${id}-${index}-bubble`}
```

Comment:
I don't think index is what you want here ... is there a better prop you could use?

Reply:
Hmm yea `item.data.key` would be better actually, which would be the titles on the bubbles like `AWS` or `Okta`. Thanks 👍